casadm/extended_err_msg.c

```diff
@@ -164,7 +164,7 @@ struct {
 	},
 	{
 		OCF_ERR_INVAL_CACHE_DEV,
-		"Device does not meet requirements."
+		"Device does not meet requirements. Check kernel dmesg for details"
```

Comment:
It would be good if we used same wording in both `KCAS_ERR_DETACHED` and here - they both direct users to check dmesg.